c-marshalling/src/lib.rs

```diff
@@ -276,8 +272,14 @@ impl<T: IntoRawConversion, E: IntoRawConversion> IntoRawConversion for Result<T,
 
     fn into_raw(self) -> Result<Self::Raw, Error> {
         Ok(match self {
-            Ok(value) => CMutResult { ok: box_into_ptr(value)?, err: ::std::ptr::null_mut() },
-            Err(value) => CMutResult { ok: ::std::ptr::null_mut(), err: box_into_ptr(value)? },
+            Ok(value) => CMutResult {
+                ok: box_into_ptr(value)?,
+                err: ::std::ptr::null_mut(),
```

Comment:
nit: remove the `::` prefix as well

Reply:
Done, and I should have removed all `::` prefix.